service/src/test/java/bio/terra/workspace/service/iam/ControlledResourceSamPolicyBuilderTest.java

```diff
@@ -0,0 +1,134 @@
+package bio.terra.workspace.service.iam;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.junit.jupiter.api.Assertions.*;
+
+import bio.terra.workspace.common.BaseSpringBootUnitTest;
+import bio.terra.workspace.common.exception.InternalLogicException;
+import bio.terra.workspace.common.utils.GcpUtils;
+import bio.terra.workspace.service.iam.model.ControlledResourceIamRole;
+import bio.terra.workspace.service.resource.controlled.model.ControlledResourceCategory;
+import bio.terra.workspace.service.workspace.model.WsmApplication;
+import bio.terra.workspace.service.workspace.model.WsmWorkspaceApplication;
+import org.broadinstitute.dsde.workbench.client.sam.model.CreateResourceRequestV2;
+import org.junit.jupiter.api.Test;
+
+class ControlledResourceSamPolicyBuilderTest extends BaseSpringBootUnitTest {
+
+  @Test
+  void addPolicies_userPrivate() {
+    var policyBuilder =
+        new ControlledResourceSamPolicyBuilder(
+            ControlledResourceIamRole.EDITOR,
+            "fake@example.com",
+            ControlledResourceCategory.USER_PRIVATE,
+            null);
+    var request = new CreateResourceRequestV2();
+
+    policyBuilder.addPolicies(request);
+
+    assertThat(
+        request.getPolicies().get(ControlledResourceIamRole.OWNER.toSamRole()).getMemberEmails(),
+        contains(GcpUtils.getWsmSaEmail()));
+    assertThat(
```

Comment:
So this would be the case of a Cromwell App being owned by Leo, but the user having Editor permissions?

Reply:
Though this is a USER_PRIVATE resource, so why does Leo get owner permissions on that?